example/index.tsx

```diff
@@ -37,47 +29,23 @@ const App: React.FC = () => {
     }
   });
 
-  // Gravity force for the opacity demo.
-  gravity1D({
+  const [propsBackground] = useGravity<HTMLDivElement>({
+    from: { background: "rgba(100, 0, 100, 0.5)" },
+    to: { background: "rgba(100, 200, 200, 1)" },
```

Comment:
🎉 We can interpolate colors! This _should_ also handle `rgb`, `hsl`, and HEX as well, but I'm going to write unit tests to make sure.